gv.c

```diff
@@ -3378,44 +3378,149 @@ Perl_try_amagic_un(pTHX_ int method, int flags) {
 
 
 /*
-=for apidoc amagic_find
+=for apidoc amagic_applies
 
-Check C<sv> for the overloaded (active magic) operation C<method>, and
-return the C<CV *> or C<NULL>.
+Check C<sv> to see if the overloaded (active magic) operation C<method>
+applies to it. If the sv is not SvROK or it is not an object then returns
+false, otherwise checks if the object is blessed into a class supporting
+overloaded operations, and returns true if a call to amagic_call() with
+this SV and the given method would trigger an amagic operation, including
+via the overload fallback rules or via nomethod. Thus a call like:
 
-C<method> is one of the values found in F<overload.h>.
+    amagic_has(sv, string_amg, AMG_unary)
```

Comment:
Looks like "amagic_has" should be "amagic_applies" here.

Reply:
Fixed, thanks.